lexer/gen/golang/acttab.go

```diff
@@ -65,6 +66,8 @@ type action struct {
 }
 
 const actionTableSrc = `
+// generated by gocc; DO NOT EDIT.
```

Comment:
I agree.

> On 07 Feb 2016, at 17:13, Walter Schulze <notifications@github.com> wrote:
> 
> In lexer/gen/golang/acttab.go:
> 
> > @@ -65,6 +66,8 @@ type action struct {
> >  }
> >  
> >  const actionTableSrc = `
> > +// generated by gocc; DO NOT EDIT.
> 
> I think since its an internal API no one should really care if it changes.
> But I guess @goccmack should give his opinion.
> 
> —
> Reply to this email directly or view it on GitHub.
>